flink-runtime/src/main/java/org/apache/flink/runtime/metrics/MetricNames.java

```diff
@@ -71,6 +71,10 @@ public static String currentInputWatermarkName(int index) {
     public static final String TASK_IDLE_TIME = "idleTimeMs" + SUFFIX_RATE;
     public static final String TASK_BUSY_TIME = "busyTimeMs" + SUFFIX_RATE;
     public static final String TASK_BACK_PRESSURED_TIME = "backPressuredTimeMs" + SUFFIX_RATE;
+    public static final String TASK_SOFT_BACK_PRESSURED_TIME =
+            "softBackPressuredTimeMs" + SUFFIX_RATE;
+    public static final String TASK_HARD_BACK_PRESSURED_TIME =
+            "hardBackPressuredTimeMs" + SUFFIX_RATE;
```

Comment:
I don't fully sure about `soft` and `hard` prefixes. In general, it tells nothing that they are responsible for. Unfortunately, I can not come up with a better idea(`blocked*`, `nonBlocked*` or `writingRecordBlockedTime`, `writingRecordAwaitTime`)

Reply:
Yeah, I get your point.